packages/docs-app/src/examples/core-examples/tabsExample.tsx

```diff
@@ -160,6 +133,49 @@ export class TabsExample extends React.PureComponent<ExampleProps, TabsExampleSt
                     <TabsExpander />
                     <InputGroup fill={true} type="text" placeholder="Search..." />
                 </Tabs>
+                <Divider style={{ margin: "20px 0", width: "100%" }} />
+                <H4>Tabs with separately rendered panels, controlled mode</H4>
+                <Switch checked={this.state.fill} label="Fill height" onChange={this.toggleFill} />
+                <div className={Classes.SECTION}>
+                    <Navbar>
+                        <Navbar.Group>
+                            <Navbar.Heading>
+                                Page: <strong>{this.state.navbarTabId}</strong>
+                            </Navbar.Heading>
+                        </Navbar.Group>
+                        <Navbar.Group align={Alignment.RIGHT}>
+                            <Tabs
+                                animate={this.state.animate}
+                                fill={this.state.fill}
+                                id={NAVBAR_PARENT_ID}
+                                large={this.state.large}
+                                onChange={this.handleNavbarTabChange}
+                                selectedTabId={this.state.navbarTabId}
+                            >
+                                <Tab id="Home" title="Home" icon={this.state.showIcon ? "home" : undefined} />
+                                <Tab id="Files" title="Files" icon={this.state.showIcon ? "folder-open" : undefined} />
+                                <Tab
+                                    id="Builds"
+                                    title="Builds"
+                                    icon={this.state.showIcon ? "build" : undefined}
+                                    tagContent={this.state.showTags ? 4 : undefined}
+                                    tagProps={{ round: this.state.useRoundTags }}
+                                />
+                            </Tabs>
+                        </Navbar.Group>
+                    </Navbar>
+                    <TabPanel
+                        id={this.state.navbarTabId}
+                        selectedTabId={this.state.navbarTabId}
```

Comment:
lol I guess I didn't see this coming where `id` could just also be dynamic 🤷

that's fine I can't think of an easy way to prevent this and maybe it is okay. if we ever actually see an accessibility issue with changing the id dynamically we can update the docs at that time but no need to go searching for if this would be an issue